CHANGES.rst

```diff
@@ -1,40 +1,66 @@
-Changelog
-=========
+Change History
+--------------
+
+.. _vunreleased:
+
+unreleased
+^^^^^^^^^^
+
+- Bring repo up to Pylons Project standards.
+  See https://github.com/Pylons/pyramid_mailer/pull/89
+
+- Optionally include bcc information on send.
+  See https://github.com/Pylons/pyramid_mailer/pull/87
+
+.. _v0.15.1:
 
 0.15.1 (2016-12-13)
--------------------
+^^^^^^^^^^^^^^^^^^^
```

Comment:
For taking all the time to change the headings, why did you select something that is unlike the other repos? I expect new versions to use `====` and subsections in a release to use `----`.

Reply:
`docs/index.rst` includes `CHANGES.rst`. I can't remember which other Pylons repo did this, but I copied that pattern. Alternatively I could create a new file, say `docs/changes.rst`, that has only an include directive, and include it in a `toctree` in `docs/index.rst`.

Reply:
Fixed per @tseaver's suggestion above in 8b432260d629604a5db7160d12b66d1a508a1a47